src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -358,6 +359,7 @@ OperatorsBridge::OperatorsBridge() {
     REGISTER_OPERATOR("If", 1, if_op);
     REGISTER_OPERATOR("ImageScaler", 1, image_scaler);
     REGISTER_OPERATOR("InstanceNormalization", 1, instance_norm);
+    REGISTER_OPERATOR("IsFinite", 1, is_finite)
```

Comment:
Based on ONNX documentation, IsFinite is not an ONNX operator. Maybe it shouldn't be registered?

Reply:
done